composer/utils/run_directory.py

```diff
@@ -1,5 +1,42 @@
 # Copyright 2021 MosaicML. All Rights Reserved.
 
+"""The run directory stores artifacts created during training.
+
+Callbacks, loggers, the checkpointer, and the profiler can write files to and read files from the run
+directory. For example,
+
+* The :class:`~composer.trainer.checkpoint.CheckpointSaver` can save checkpoints inside the run directory.
+* The :class:`~composer.loggers.file_logger.FileLogger` can save log files inside the run directory.
+* The :class:`~composer.profiler.profiler.Profiler` can save profiling traces inside the run directory.
+* The :class:`~composer.callbacks.run_directory_uploader.RunDirectoryUploader` backs up files written to the
+    run directory to cloud object storage.
+* The :class:`~composer.loggers.wandb_logger.WandBLogger` can store files in the run directory as artifacts.
+
+Run Directory Location
+**********************
+
+If using the :ref:`composer launch script for distributed training <distributed-training>`,
+the run directory can be specified using the ``--run_directory`` command line flag. For example:
+
+.. console::
+
+    composer --run_directory /path/to/run/directory -n 8 path/to/training/script.py
+
+Otherwise, if launching python directly, the run directory can be specified via the ``COMPOSER_RUN_DIRECTORY`` environment variable.
+(The :ref:`composer launch script <distributed-training>` sets this variable for you.)
+
+If not specified, then the run directory defaults to ``f"./runs/{datetime.datetime.now().isoformat()}"``,
```

Comment:
this renders weirdly. not sure if there's a way to start the path on a new line.